web/src/components/pnlData/index.js

```diff
@@ -45,7 +51,7 @@ export const PnlData = () => {
       ws.removeEventListener('close', handleClose)
       ws.close()
     }
-  }, [ws])
```

Comment:
Dependency array has been updated - The `WebSocket` connection will be established when the component mounts and cleaned up when it unmounts.